docs/design/kotlin-smithy-sdk.md

```diff
@@ -249,12 +227,6 @@ val mystruct2 = mystruct.copy {
 println(mystruct2)
 // MyStruct(foo=copied, bar=12, baz=Baz(quux=foo), yesno=raw)
 
-// Java interop builder
-val mystruct3 = MyStruct.builder()
-        .foo("fooey")
-        .bar(12)
-        .build()
-
 println(mystruct3)
 // MyStruct(foo=fooey, bar=12, baz=null, yesno=null)
 ```
```

Comment:
**Correctness**: This file still contains the following phrase which should now be removed:

> For each structure shape builders are provided for constructing immutable objects using both a Kotlin DSL approach as well as a traditional builder for Java interop

Reply:
good catch